packages/contract-addresses/addresses.json

```diff
@@ -13,13 +13,14 @@
         "dutchAuction": "0x0000000000000000000000000000000000000000",
         "coordinatorRegistry": "0x45797531b873fd5e519477a070a955764c1a5b07",
         "coordinator": "0x38a795580d0f687e399913a00ddef6a17612c722",
+        "libTransactionDecoder": "0x5f20e82643ce007d87692eb1b3d3fc059588b224",
         "multiAssetProxy": "0xef701d5389ae74503d633396c4d654eabedc9d78",
         "staticCallProxy": "0x3517b88c19508c08650616019062b898ab65ed29",
         "erc1155Proxy": "0x7eefbd48fd63d441ec7435d024ec7c5131019add",
         "zrxVault": "0xba7f8b5fb1b19c1211c5d49550fcd149177a5eaf",
         "staking": "0x2a17c35ff147b32f13f19f2e311446eeb02503f3",
         "stakingProxy": "0xa26e80e7dea86279c6d778d702cc413e6cffa777",
-        "devUtils": "0x5f53f2aa72cb3a9371bf3c58e8fb3a313478b2f4",
+        "devUtils": "0xf8ba284450dcd7518bc5c68a4d93b60f734c3d72",
```

Comment:
Reminder: The `devUtils` addresses will need to be updated to reflect the changes introduced because of @mzhu25's feedback